src/l2_tracker/events.h

```diff
@@ -16,13 +16,21 @@ using namespace std::literals;
 namespace eth::event {
 
 struct L2StateChange {
-    uint64_t l2_height = 0;
+    uint64_t chain_id;
```

Comment:
It's probably not a problem but we will repeat chain IDs for our test networks because we will re-deploy onto Sepolia Arbitrum, but those are only for our test networks. So be it.

Reply:
That shouldn't be a problem because we still have the l2_height here which will still differ across re-deployments.